nabweb/static/nabweb/weather_animations.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+  <meta charset="UTF-8">
+  <title>Nabaztag :: Weather forecast visual animations</title>
+  <link rel="stylesheet" href="css/bootstrap.min.css">
+  <style>
+    table td {
+      text-align: center;
+      vertical-align: middle;
+    }
+    img {
+      max-width: 100%;
+      max-height: 100%;
+    }
+  </style>
+</head>
+
+<body>
+  <header class="navbar navbar-expand-lg navbar-light bg-light">
+     <span class="navbar-brand mb-0 h1">Nabaztag help</span>
+  </header>
+
+  <div class="container-fluid mt-3">
+    <div class="row justify-content-center">
+      <div class="col-xl-8 col-lg-10 col-md-10 mb-3">
+
+        <div class="card" id="nabweatherd-animations">
+          <div class="card-header">
+            <h5 class="card-title">The weather forecast visual animations</h5>
+          </div>
+
+          <div class="card-body">
+            <table>
+              <tr>
+                <td>Sunshine<br>Soleil</td>
+                <td><img src="images/soleil.gif"></td>
```

Comment:
I suggest to use english files naming.

Reply:
Nope. I kept the original file names from https://www.tagtagtag.fr/weather.html on purpose.